src/aiida_quantumespresso/cli/utils/validate.py

```diff
@@ -60,11 +60,11 @@ def validate_hubbard_parameters(structure, parameters, hubbard_u=None, hubbard_v
 
         try:
             hubbard_file = load_node(pk=hubbard_file_pk)
-        except exceptions.NotExistent:
-            ValueError(f'{hubbard_file_pk} is not a valid pk')
+        except exceptions.NotExistent as exc:
+            raise ValueError(f'{hubbard_file_pk} is not a valid pk') from exc
         else:
             if not isinstance(hubbard_file, SinglefileData):
-                ValueError(f'Node<{hubbard_file_pk}> is not a SinglefileData but {type(hubbard_file)}')
+                raise ValueError(f'Node<{hubbard_file_pk}> is not a SinglefileData but {type(hubbard_file)}')
```

Comment:
These are actually bugs, so we should probably update the tests for these. Probably in a separate PR. @bastonero could you take care of this please?

Reply:
Sure! Next days I am on holidays, so probably won't be able to prepare the PR. I will look at this next Tuesday. So just adding tests for the `hubbard_file` and these lines?